### PR TITLE
Substituted var with let/const; new lesson scope-2

### PR DESCRIPTION
**Pull request by huberv**
Hi all,
I modernized the lessons somewhat:
* Substituted the var keyword with const/let. Newcomers should learn the modern stuff right from the start.
* Adapted the description of the variable lesson (Enlish version only).
* Added new lesson scope-2 (English only).
* Adapted text of lesson scope (now sets the stage to the new lesson scope-2) (English only).

I'd be glad if you would incorporate the changes in order to allow newbees to use a modern JavaScript style right from the start.

Unfortunately I'm not aware of how the process for triggering translations of the other currently supported languages is. Please advise on this topic.

### CODE REVIEW

**Comment by AnshulMalik** [comment]
Hey @huberv This is great!
This PR contains lots of changes, it'd be great if we can create separate PRs for smaller changes :)

**Comment by huberv** [comment]
Hi @AnshulMalik, that's definitely true, but from my point of view it doesn't make sense to split up the change from "var" to "const" or "let" into different pull requests => you'd have to accept all of them in order to get a consistent update. 

I know the changes are in lots of files, but each individual change is quite small...

**Comment by AnshulMalik** [comment]
Looks like it needs a rebase, then we are good to go. @huberv

File: problems/scope/problem.md [hunk 1]
```diff
@@ -38,19 +38,19 @@ Create a file named `scope.js`.
...
-	var b = 5, c = 6;
+	const b = 5, c = 6;
 
 	(function secondFunction(){
-		var b = 8;
+		const b = 8;
 
 		(function thirdFunction(){
-			var a = 7, c = 9;
+			vconstar a = 7, c = 9;
```
Comment (AnshulMalik) [inline problems/scope/problem.md]: typo

Comment (huberv) [reply problems/scope/problem.md]: I fixed the type with a new commit

**Comment by huberv** [comment]
@AnshulMalik Thanks for the review! I fixed the typo you found and updated the pull request according to the procedure described here: https://help.github.com/articles/syncing-a-fork/ (I'm not in love with rebasing repos). 
Please double-check again.

**Comment by lupomontero** [comment]
Hi there, any news with regards to this PR? I think it would make a lot of sense to include these changes and start introducing more _modern_ features. Anything I can do to help?

**Comment by itzsaga** [comment]
@lupomontero well, currently there's a conflict on this PR. So resolving that would be a first step.

**Comment by huberv** [comment]
@lupomontero @itzsaga 
Hi! I just resolved the conflict.

I'd appreciate if my pull request would finally make it into the master. 

Because the pull request has been waiting for quite some time I suggest that somebody pulls from the master to my pull request, does a code review and then merges the pull request into the master.

**Comment by ledsun** [comment]
#265 is the same change as this PR.

**Pull request closed without merging**